wallet/src/Dscp/Wallet/KeyStorage.hs

```diff
@@ -0,0 +1,63 @@
+module Dscp.Wallet.KeyStorage
+       ( getAccounts
+       , addAccount
+       ) where
+
+import Data.Aeson (eitherDecode, encode)
+import Data.Aeson.TH (defaultOptions, deriveJSON)
+import Dscp.Core (mkAddr)
+import System.Directory (doesFileExist)
+import System.FileLock (SharedExclusive (..), withFileLock)
+
+import qualified Data.ByteString.Lazy as LBS
+
+import Dscp.Wallet.Face
+
+data Storage = Storage
+    { accounts :: [StorageAccount]
+    }
+
+data StorageAccount = StorageAccount
+    { secretKey :: Encrypted SecretKey
+    , publicKey :: PublicKey
+    }
+
+deriveJSON defaultOptions ''Storage
+deriveJSON defaultOptions ''StorageAccount
+
+storageFileName :: FilePath
```

Comment:
Should be configurable probably.

Reply:
Will be configurable later, when I adapt `Ariadne.Config` here

Reply:
@volhovm is cooking some common approach to configs for dscp, you might wish to consult with him first (everything for the sake of uniformity!)

Reply:
As these particular executables will be heavily based on Ariadne and mostly independent from the rest of the codebase, the separate configuration won't be so bad.

Anyway, Ariadne's configuration is still WIP, so we'll see.

Also, I'm not planning alpha version of the wallet to be very configurable.